src/Console/Installer.php

```diff
@@ -156,7 +156,17 @@ public static function setFolderPermissions($dir, $io)
             }
         };
 
-        $worldWritable = bindec('0000000111');
+        $worldWritable = 007;
+        if ($worldWritable === 007) {
```

Comment:
Since the assignment for this is one line above it - this condition will always be true.